bin/.gitignore

```diff
@@ -0,0 +1,18 @@
+classes/
```

Comment:
My original comment was probably wrong, sorry about that.

Why do you add this file at all ?
It should already be there in the top directory.